crates/re_log_types/src/data_table.rs

```diff
@@ -347,20 +372,66 @@ impl DataTable {
                 .filter_map(|rows| rows[i].clone() /* shallow */);
 
             DataRow::from_cells(
-                row_id[i],
-                timepoint[i].clone(),
-                entity_path[i].clone(),
-                num_instances[i],
+                col_row_id[i],
+                col_timepoint[i].clone(),
+                col_entity_path[i].clone(),
+                col_num_instances[i],
                 cells,
             )
         })
     }
 
+    #[inline]
+    pub fn into_rows(self) -> impl Iterator<Item = DataRow> {
+        let Self {
+            table_id: _,
+            col_row_id,
+            col_timepoint,
+            col_entity_path,
+            col_num_instances,
+            columns,
+        } = self;
+
+        use itertools::Itertools as _;
+
+        let col_row_id = col_row_id.into_iter();
+        let col_timepoint = col_timepoint.into_iter();
+        let col_entity_path = col_entity_path.into_iter();
+        let col_num_instances = col_num_instances.into_iter();
+
+        let mut columns = columns
+            .into_values()
+            .map(|column| column.0.into_iter())
+            .collect_vec();
+        let rows = std::iter::from_fn(move || {
+            let mut next = Vec::with_capacity(columns.len());
```

Comment:
this could end up wasting a lot of ram for sparse tables

Reply:
this entire thing is actually pointless now that virtual calls are gone -> :wastebasket: